src/classes/whdata.py

```diff
@@ -118,31 +121,20 @@ def create_views_df_json(data):
 
             log.info('%7d total records processed', total)
             log.info('%7d ads ignored, %d were surveys',
-                total - views_df.shape[0], survey_count)
+                     total - views_df.shape[0], survey_count)
             log.info('%7d views', views_df.shape[0])
 
         return views_df
 
-    #div.outer-cell
```

Comment:
as a general rule of thumb, if you want to retain commented out code, you should leave an explanation as to why you want to retain it. Otherwise, just remove it once you're sure you don't need it anymore.

I couldn't understand what you were trying to say with this comment, so I just removed it.